frontend/src/components/common/player/player.tsx

```diff
@@ -0,0 +1,124 @@
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import styles from './styles.module.scss';
+
+type Props = {
+  src: string;
+  onClickPrevious?: () => void;
+  onClickNext?: () => void;
+};
+
+const MILLISECONDS_IN_SECOND = 1000;
+const SKIP_IN_MILLISECONDS = 15000;
+
+const Player: React.FC<Props> = ({ src, onClickPrevious, onClickNext }) => {
+  return (
+    <AudioPlayer
+      src={src}
+      showSkipControls={true}
+      progressJumpSteps={{
+        backward: SKIP_IN_MILLISECONDS,
+        forward: SKIP_IN_MILLISECONDS,
+      }}
+      customAdditionalControls={[]}
+      onClickNext={onClickNext}
+      onClickPrevious={onClickPrevious}
+      customIcons={{
+        play: (
+          <div
+            className={[
+              styles.playerCommonBtn,
+              styles.playerMainBtn,
+              styles.playerPlayBtn,
+            ].join(' ')}
+          >
+            <svg viewBox="0 0 14 20" xmlns="http://www.w3.org/2000/svg">
+              <path d="M13.5383 9.38518L1.23076 0.154505C0.999989 -0.0208782 0.686147 -0.0485703 0.426149 0.0821977C0.164614 0.212966 0 0.479117 0 0.769883L0 19.2312C0 19.522 0.164614 19.7882 0.424611 19.9189C0.533841 19.9728 0.652301 20.0005 0.769223 20.0005C0.932298 20.0005 1.09537 19.9482 1.23076 19.8466L13.5383 10.6159C13.7322 10.4713 13.846 10.2421 13.846 10.0006C13.846 9.75903 13.7322 9.5298 13.5383 9.38518Z" />
+            </svg>
```

Comment:
```suggestion
            <svg viewBox="0 0 14 20" xmlns="http://www.w3.org/2000/svg">
              <path d="M13.5383 9.38518L1.23076 0.154505C0.999989 -0.0208782 0.686147 -0.0485703 0.426149 0.0821977C0.164614 0.212966 0 0.479117 0 0.769883L0 19.2312C0 19.522 0.164614 19.7882 0.424611 19.9189C0.533841 19.9728 0.652301 20.0005 0.769223 20.0005C0.932298 20.0005 1.09537 19.9482 1.23076 19.8466L13.5383 10.6159C13.7322 10.4713 13.846 10.2421 13.846 10.0006C13.846 9.75903 13.7322 9.5298 13.5383 9.38518Z" />
            </svg>
```
`import {ReactComponent as PlayIcon} from './assets/img/play.svg';`
do it for all

Reply:
We have an Icon folder in assets for this purpose